src/components/listing-search-page/ListingSearch/index.jsx

```diff
@@ -0,0 +1,71 @@
+import room from 'assets/images/living-room.png';
+import Map from 'components/shared/Map';
+import React from 'react';
+import 'slick-carousel/slick/slick-theme.css';
+import 'slick-carousel/slick/slick.css';
+import MapCard from '../MapCard';
+import ListingSearch from '../SearchResults';
+import { ListingPage, MapContainer } from './listing-search.style';
+
+const fakeData = [
+  {
+    id: 1,
+    image: room,
+    listingBy: 'seller',
+    location: {
+      lat: 35.372,
+      lng: 135.83,
+    },
+    locationStreet: '107 Rose Dr, Knoxville, TN 37918',
+    price: '$750-$800',
```

Comment:
```suggestion
    price: '750-800',
```
In case new currency is added.